content/hi/docs/tasks/tools/install-kubectl-macos.md

```diff
@@ -0,0 +1,248 @@
+   ```
+
+   और Apple Silicon macOS के लिए, टाइप करें:
+
+   ```bash
+   curl -LO "https://dl.k8s.io/release/{{< param "fullversion" >}}/bin/darwin/arm64/kubectl"
+   ```
+
+   {{< /note >}}
+
+1. बाइनरी को मान्य करें (ऐच्छिक))
+
+   kubectl चेकसम फाइल डाउनलोड करें:
+
+   {{< tabs name="download_checksum_macos" >}}
+   {{< tab name="Intel" codelang="bash" >}}
+   curl -LO "https://dl.k8s.io/release/$(curl -L -s https://dl.k8s.io/release/stable.txt)/bin/darwin/amd64/kubectl.sha256"
+   {{< /tab >}}
+   {{< tab name="Apple Silicon" codelang="bash" >}}
+   curl -LO "https://dl.k8s.io/release/$(curl -L -s https://dl.k8s.io/release/stable.txt)/bin/darwin/arm64/kubectl.sha256"
+   {{< /tab >}}
+   {{< /tabs >}}
+  
+   चेकसम फ़ाइल के विरुद्ध kubectl बाइनरी को मान्य करें:
+
+   ```bash
+   echo "$(<kubectl.sha256)  kubectl" | shasum -a 256 --check
+   ```
+
+   यदि मान्य है, तो आउटपुट है:
+
+   ```console
+   kubectl: OK
+   ```
+
+   अगर चेक फेल हो जाता है, `sha256` गैर-शून्य स्थिति के साथ बाहर निकलता है और इस आउटपुट के समान प्रिंट करता है:
+
+   ```bash
+   kubectl: FAILED
+   shasum: WARNING: 1 computed checksum did NOT match
+   ```
+
+   {{< note >}}
+   बाइनरी और चेकसम का एक ही संस्करण डाउनलोड करें.
+   {{< /note >}}
+
+1. kubectl बाइनरी को एक्सेक्यूट करने योग्य बनायें.
+
+   ```bash
+   chmod +x ./kubectl
+   ```
+
+1. kubectl बाइनरी को अपने सिस्टम पर `PATH` फ़ाइल स्थान पर ले जाएँ.
+
+   ```bash
+   sudo mv ./kubectl /usr/local/bin/kubectl
+   sudo chown root: /usr/local/bin/kubectl
+   ```
+
+   {{< note >}}
+   सुनिश्चित करें कि `/usr/local/bin` आपके पाथ एनवायरनमेंट वेरिएबल में है.
+   {{< /note >}}
+
+1. यह सुनिश्चित करने के लिए परीक्षण करें कि आपके द्वारा इंस्टॉल किया गया संस्करण अद्यतित है:
+
+   ```bash
+   kubectl version --client
+   ```
+
+### macOS पर Homebrew के साथ इंस्टॉल करें
+
+यदि आप macOS पर हैं और [Homebrew](https://brew.sh/) पैकेज मैनेजर का उपयोग कर रहे हैं, आप Homebrew के साथ kubectl इंस्टॉल कर सकते हैं.
+
+1. इंस्टालेशन कमांड रन करें:
+
+   ```bash
+   brew install kubectl 
+   ```
+
+   or
+
+   ```bash
+   brew install kubernetes-cli
+   ```
+
+1. यह सुनिश्चित करने के लिए परीक्षण करें कि आपके द्वारा इंस्टॉल किया गया संस्करण अद्यतित है:
+
+   ```bash
+   kubectl version --client
+   ```
+
+### macOS पर Macports के साथ इंस्टॉल करें
+
+यदि आप macOS पर हैं और [Macports](https://macports.org/) पैकेज मैनेजर का उपयोग कर रहे हैं, आप Macports के साथ kubectl इंस्टॉल कर सकते हैं.
+
+1. इंस्टालेशन कमांड रन करें:
+
+   ```bash
+   sudo port selfupdate
+   sudo port install kubectl
+   ```
+
+1. यह सुनिश्चित करने के लिए परीक्षण करें कि आपके द्वारा इंस्टॉल किया गया संस्करण अद्यतित है:
+
+   ```bash
+   kubectl version --client
+   ```
+
+## kubectl कॉन्फ़िगरेशन सत्यापित करें
+
+{{< include "included/verify-kubectl.md" >}}
+
+## वैकल्पिक Kubectl कॉन्फ़िगरेशन और प्लगइन्स
+
+### शेल ऑटोकम्प्लेशन सक्षम करें
+
+kubectl Bash और Zsh के लिए ऑटोकम्प्लेशन का सपोर्ट प्रदान करता है, जो आपको बहुत सारी टाइपिंग बचा सकता है.
```

Comment:
```suggestion
kubectl Bash और Zsh के लिए ऑटोकम्प्लेशन का सपोर्ट प्रदान करता है, जो आपका काफी समय बचा सकता है।
```

Reply:
I think we can replace it with another phrase which gives the same sense as `आपको बहुत सारी टाइपिंग बचा सकता है` sounds a bit off.